ci/build_wheel_cudf_streaming.sh

```
@@ -0,0 +1,61 @@
+source rapids-init-pip
+
+package_name="cudf-streaming"
+package_dir="python/cudf_streaming"
+dependency_file_key_suffix="cudf_streaming"
+
+RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen "${RAPIDS_CUDA_VERSION}")"
+
+# Downloads libcudf_streaming wheel from this current build,
+# then ensures 'cudf_streaming' wheel builds always use the 'libcudf_streaming' just built in the same CI run.
+LIBCUDF_STREAMING_WHEELHOUSE=$(RAPIDS_PY_WHEEL_NAME="libcudf_streaming_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-github cpp)
+echo "libcudf-streaming-${RAPIDS_PY_CUDA_SUFFIX} @ file://$(echo ${LIBCUDF_STREAMING_WHEELHOUSE}/libcudf_streaming_*.whl)" >> "${PIP_CONSTRAINT}"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)